src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -108,11 +108,15 @@ def _reset_tracking(self):
   def apply_extension_fields(self, extension_fields):
     """Applies _ext_ prefixed extension fields."""
     if extension_fields.get('_ext_collaborators'):
+      print('google_issue_tracker: In apply_extension_fields for '
```

Comment:
What are these print statements for?
We can't read them when they are run on google cloud.
Is it posisble you want logs.log

Reply:
I mentioned this in the PR-
"Used print instead of logs.log because I am not sure how the logger is configured for ClusterFuzz cloud logs. print should show up in stdout."

Normally GCP automatically collects logs from stdout and stderr so print should work. But I can change to logs.log if we are not sure.

Reply:
Yes please switch. Sorry for not reading.

Reply:
NP. Done!